VERSION

```diff
@@ -1 +1 @@
-1.5.2
+1.6.0
```

Comment:
Not sure, if the plugin version needs to match foglamp version. Plugin version can probably be independent.